### ISSUE TITLE
2 blocks from WDAC, on WHH Light 

### ISSUE THREAD

**Issue by shmu26** [AndyFul/Hard_Configurator#21]
I got two blocks after enabling WDAC
`******** WDAC blocked events for EXE and DLL files ********
***********************************************************

Event[0]:
Event Id = 3077
Local Time:  2024/04/08 19:58:55
Attempted Path = C:\Windows\System32\wsl.exe
Parent Process = C:\Program Files\WindowsApps\CanonicalGroupLimited.Ubuntu_2204.3.49.0_x64__79rhkp1fndgsc\ubuntu.exe
PolicyName = UserSpace Lock
UserWriteable = false

***********************************************************
***********************************************************

Event[1]:
Event Id = 3077
Local Time:  2024/04/08 19:56:10
Attempted Path = C:\Windows\System32\wsl.exe
Parent Process = C:\Program Files\WindowsApps\CanonicalGroupLimited.Ubuntu_2204.3.49.0_x64__79rhkp1fndgsc\ubuntu.exe
PolicyName = UserSpace Lock
UserWriteable = false

***********************************************************
***********************************************************

Event[2]:
Event Id = 3077
Local Time:  2024/04/08 19:42:22
Attempted Path = C:\Windows\SysWOW64\wbem\WMIC.exe
Parent Process = C:\Windows\SysWOW64\cmd.exe
PolicyName = UserSpace Lock
UserWriteable = false

***********************************************************
***********************************************************`

**Comment by AndyFul** [comment]
Yes, two LOLBins (WMIC.exe and wsl.exe) are blocked in WDAC policy by Microsoft's recommended rules. These LOLBins could be sed to bypass WDAC.

**Comment by shmu26** [comment]
> Yes, two LOLBins (WMIC.exe and wsl.exe) are blocked via WDAC policy by Microsoft's recommended rules.
But do you have any idea why cmd would spawn WMIC in normal computer usage of Windows 11? The only thing I regularly use cmd for is to shut down wsl, via the `command wsl --shutdown`

**Comment by AndyFul** [comment]
This is uncommon behavior. Probably some application tries to run a script. Did you look at the SWH Blocked events?
